### ISSUE TITLE
Upgraded Router - Can't Connect

### ISSUE THREAD

**Issue by ThomMark** [koying/SPMC#1081]
Hi Peeps

I have upgraded my Virgin Media router, I have named it exactly the same as the old one, yet my SPMC can not connect to the folders I made to share my movies etc.
When I use the PC, it connects to the folder and streams absolutely fine, also the internet is also working fine.

Workgroup is still the same
Router name is the same
Wifi username and password are the same

Any help or guidance will be gratefully received.

Thanks
Mark

**Comment by ziggy73701** [comment]
Going to need a little more info... where are the shares located??? Nas.. Windows’s box?? Did internal IP change?

**Comment by ThomMark** [comment]
Hi and thanks for your reply, the shares are on a Western Digital NAS drive,  I am not sure about the IP address. If I put my other router back on, it all works. 
Then if  I switch back, it stops, although everything else is fine.

**Comment by ziggy73701** [comment]
What is the NAS drive plugged in to?
Are you able to see them from any other device?

**Comment by ThomMark** [comment]
Hi mate, sorry I was at my kids parent's evening.

OK so here is my rig.

Back room: Virgin Media Broadband router.
   Ethernet cable from that into front room, connect's into a 4 port RJ45 hub.
      Connected to that hub is: PS3, Western Digital NAS Drive, Samsung Smart TV, Hardwired Amazon fire stick.

On that set up, everything works perfect.

By replacing the Virgin Media router in the back room, I still have the one cable going to the other room.
The PS3, Samsung Smart TV can both see the NAS drive and stream from it.
Using my firestick and opening SPMC, the media can not been seen "Connection failed".

My laptop and PC can still access everything, and everything else is working perfect.

I am therefor guessing this points back to your previous mention of the IP address?

Thanks for your help so far
Mark

**Comment by ziggy73701** [comment]
Yeah am going DNS / IP filth..... stupid question... but have you restarted all devices?

**Comment by ThomMark** [comment]
Yes mate, powered down and started devices one at a time.

Been at it for hours, I bet it's such a simple switch/flick of a button to get it all back on lol.

Does SPMC look for the an IP address, or just the NAS drive on the same network?

**Comment by ziggy73701** [comment]
Sorry mate I had to check!!!

SPMC will use SMB 3 which won’t allow you to browse. The fact it works on 1 is a little odd!!

Am guessing everything else on the fire stick works ok?? You get in to amazon store or you can use YouTube etc?

**Comment by ThomMark** [comment]
Yes, everything else works, I have IPTV which is also working on it.

I switched back to my other router just then , and it fired up no problems, all my folders and contents can be seen and playing.

???? Confused.com

**Comment by ziggy73701** [comment]
Yeah that seems proper garbage that!!!! 
My only guess... is that the old router has some kind of WINS or other DNS setting which allowed the local lookup to work. 
So... with the OLD device plugged in, go in your network setting on the fire stick and see what it says. Make a note of it all. Sorry.. I don’t have one so I don’t know what it will say. If it’s a windows laptop you have, do a ipconfig /all 
Then.. plug in the new router.. do the same.. see what’s different.

**Comment by ThomMark** [comment]
Ok buddy, Will give that a try.
Going to try my luck now with the Mrs. If I'm not back in half hour I'm in... Otherwise I'll get on this tomorrow ;)
